### PR TITLE
Documentation for the `@rotate` attribute

### PR DESCRIPTION
**Pull request by JRegimbal**
Add basic documentation to the facsimile documentation mentioning the `@rotate` attribute, which follows from [the TEI attribute of the same name](https://tei-c.org/release/doc/tei-p5-doc/en/html/ref-zone.html). This corresponds to the changes in the schema PR music-encoding/music-encoding#674.

### CODE REVIEW

**Comment by JRegimbal** [comment]
Please let me know if anything from music-encoding/music-encoding#620 should be a part of this!

File: _guidelines-v4/12-facsimilesrecordings/01-facsimiles/01-facsimileelements.md [hunk 1]
```diff
@@ -36,6 +36,11 @@ Because the coordinate space of a zone is defined relative to that of a surface,
 
 {% include mei example="facsimiles/facsimiles-sample249.xml" valid="" %}
 
+Additionally, the area represented by a {% include link elem="zone" %} element may not be normally oriented to its parent surface. The amount by which the area has been rotated clockwise in degrees is expressed by the **@rotate** attribute. By default this amount is 0&deg;.
```
Comment (bwbohl) [inline _guidelines-v4/12-facsimilesrecordings/01-facsimiles/01-facsimileelements.md:39]: The amount by which the contents of this area…

**Review by bwbohl — CHANGES_REQUESTED** [review]

File: _guidelines-v4/12-facsimilesrecordings/01-facsimiles/01-facsimileelements.md (hunk 1 above)
Comment (bwbohl) [reply _guidelines-v4/12-facsimilesrecordings/01-facsimiles/01-facsimileelements.md:39]: should be equal to the attribute description in the proposed schema PR

Comment (JRegimbal) [reply _guidelines-v4/12-facsimilesrecordings/01-facsimiles/01-facsimileelements.md]: I just added a commit trying to do this. Is it okay?

**Comment by kepper** [comment]
I think this can be safely merged, so should be done tomorrow at ODD Friday.

**Pull request merged**